verification/curator-service/ui/cypress/integration/components/NewCaseForm.spec.ts

```diff
@@ -255,6 +255,61 @@ describe('New case form', function () {
         });
     });
 
+    it('Can add places of transmission from chips', function () {
+        cy.seedLocation({
+            country: 'France',
+            geometry: { latitude: 45.75889, longitude: 4.84139 },
+            name: 'France',
+            geoResolution: 'Country',
+        });
+        cy.addSource('Test source', 'www.example.com');
+        cy.addCase({
+            country: 'France',
+            notes: 'some notes',
+            sourceUrl: 'www.example.com',
+            transmissionPlaces: ['Gym', 'Hospital'],
+        });
+        cy.addCase({
+            country: 'France',
+            notes: 'some notes',
+            sourceUrl: 'www.example.com',
+            transmissionPlaces: [
+                'Airplane',
+                'Factory',
+                'Gym',
+                'Hospital',
+                'Hotel',
+                'Office',
+            ],
+        });
+
+        cy.visit('/cases/new');
+        cy.contains('Create new COVID-19 line list case');
+        cy.get('div[data-testid="caseReference"]').type('www.example.com');
+        cy.contains('li', 'www.example.com').click();
+        cy.get('div[data-testid="location"]').type('France');
+        cy.contains('li', 'France').click();
+        cy.get('input[name="confirmedDate"]').type('2020-01-01');
+        cy.contains('Gym');
+        cy.contains('Hospital');
+        cy.contains('Airplane');
+        cy.contains('Factory');
+        cy.contains('Hotel');
```

Comment:
Do we care about the ordering, and if so, should we test it?

Reply:
No we don't care about the ordering